docs/commands/idm.md

```diff
@@ -0,0 +1,40 @@
+# `idm` command
```

Comment:
My main concern with all these pages is that they get out of date so quickly. We change commands, add new ones, and doc is always behind.

Also, most of what we have in these command pages seems to be basically the same as what a user could get from running the frodo help function for a command.

Maybe if instead we gave away some secrets or tips?

Reply:
I had the same feeling when looking at the command specific content. That is why I didn't merge the request and wanted another opinion. I agree that the cli help should be only source of that information and if needed we can make it friendlier/verbose. But, I also think that we should keep the list of supported features updated in the README. Thoughts?